cmd/archive.go

```diff
@@ -55,6 +56,12 @@ func (c *cmdArchive) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if c.skipEnv {
+		c.gs.logger.Info("environment variables will be excluded from the archive")
+
+		arc.WipeEnv()
```

Comment:
Considering that `arc.Env` is public, why do we need a new method? :confused:

Reply:
Yes, `arc.Env` is public, and we can modify it without the method. Still, using the method, we also consolidate (and document) the modification's purpose, making it more maintainable for the future, WDYT?

Reply:
I think it's fine to directly manipulate the public property. If we ever decide to make it private in the future, this code won't compile and we can make a method then. But as long as the property is private, and we only wipe it here, I don't see a need for a method.

Reply:
Okay, I'll change it